database/migrations/2016_08_18_152505_create_members_table.php

```diff
@@ -12,17 +12,15 @@ class CreateUsersTable extends Migration
      */
     public function up()
     {
-        Schema::create('users', function (Blueprint $table) {
+        Schema::create('members', function (Blueprint $table) {
```

Comment:
I think @glitchinfinity has a point, as this would break all of the current working environments. But as not many people are using it now, I think it would be wise to do it all over instead of adding another migration for table renaming, it would make things a little bit clearer